corehq/apps/app_manager/tasks.py

```diff
@@ -37,12 +37,16 @@ def make_async_build_v2(app_id, domain, version, username, allow_prune=True, com
         return copy
 
 
-def make_async_build(app, username, allow_prune=True, comment=None):
+def make_async_build(app, username, allow_prune=True, comment=None, bypass_async=False):
```

Comment:
the naming is reconsidered later 
`make_async_build` -> `make_build`
`bypass_async=False` -> `sync=True`